stdlib/3/winreg.pyi

```diff
@@ -0,0 +1,78 @@
+
+import sys
+
+from typing import Optional, Type, Union, Tuple, Any
+from types import TracebackType
+
+_KeyType = Union[PyHKEY, int]
+
+def CloseKey(hkey: _KeyType): ...
+def ConnectRegistry(computer_name: Optional[str], key: _KeyType) -> PyHKEY: ...
+def CreateKey(key: _KeyType, sub_key: Optional[str]) -> PyHKEY: ...
+def CreateKeyEx(key: _KeyType, sub_key: Optional[str], reserved: int = ..., access: int = ...) -> PyHKEY: ...
+def DeleteKey(key: _KeyType, sub_key: str) -> None: ...
+def DeleteKeyEx(key: _KeyType, sub_key: str, access: int = ..., reserved: int = ...) -> None: ...
+def DeleteValue(key: _KeyType, value: str) -> None: ...
+def EnumKey(key: _KeyType, index: int) -> str: ...
+def EnumValue(key: _KeyType, index: int) -> Tuple[str, Any, int]: ...
+def ExpandEnvironmentStrings(str: str) -> str: ...
+def FlushKey(key: _KeyType) -> None: ...
+def LoadKey(key: _KeyType, sub_key: str, file_name: str) -> None: ...
+def OpenKey(key: _KeyType, sub_key: str, reserved: int = ..., access: int = ...) -> PyHKEY: ...
+def OpenKeyEx(key: _KeyType, sub_key: str, reserved: int = ..., access: int = ...) -> PyHKEY: ...
+def QueryInfoKey(key: _KeyType) -> Tuple[int, int, int]: ...
+def QueryValue(key: _KeyType, sub_key: Optional[str]) -> str: ...
+def QueryValueEx(key: _KeyType, value_name: str) -> Tuple[Any, int]: ...
+def SaveKey(key: _KeyType, file_name: str) -> None: ...
+def SetValue(key: _KeyType, sub_key: str, type: int, value: str) -> None: ...
+def SetValueEx(key: _KeyType, value_name: str, reserved: Any, type: int, value: str) -> None: ...  # reserved is ignored
+def DisableReflectionKey(key: _KeyType) -> None: ...
+def EnableReflectionKey(key: _KeyType) -> None: ...
+def QueryReflectionKey(key: _KeyType) -> bool: ...
+
+HKEY_CLASSES_ROOT: int
+HKEY_CURRENT_USER: int
+HKEY_LOCAL_MACHINE: int
+HKEY_USERS: int
+HKEY_PERFORMANCE_DATA: int
+HKEY_CURRENT_CONFIG: int
+HKEY_DYN_DATA: int
+
+KEY_ALL_ACCESS: int
+KEY_WRITE: int
+KEY_READ: int
+KEY_EXECUTE: int
+KEY_QUERY_VALUE: int
+KEY_SET_VALUE: int
+KEY_CREATE_SUB_KEY: int
+KEY_ENUMERATE_SUB_KEYS: int
+KEY_NOTIFY: int
+KEY_CREATE_LINK: int
+
+KEY_WOW64_64KEY: int
+KEY_WOW64_32KEY: int
+
+REG_BINARY: int
```

Comment:
There seem to be a bunch more of these (e.g. `REG_LEGAL_CHANGE_FILTER`), but they're undocumented.